frontend/src/tabs/authorship.pug

```diff
@@ -77,6 +77,13 @@
               span(v-bind:title="getFileTypeBlankLineInfo(fileType)")
                 span {{ fileType }}&nbsp;{{ fileTypeLinesObj[fileType] }}&nbsp;
                 span ({{ fileTypeLinesObj[fileType] - fileTypeBlankLinesObj[fileType] }})&nbsp;
+          span &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
why the spacing is controlled by &nbsp instead of css?

Reply:
Uh this is to prevent the situation where the binary checkbox appears as the first checkbox on a new line, in which case setting left margin in css may cause unnecessary indentation of the checkbox

Reply:
I think it is possible to make this binary label automatically aligned to right. Can you give some screenshots of

1.  binary first of row with nbsp
2.  binary  first of row with css
3.  binary aligned to right

And seek comments from Prof?

Reply:
![image](https://user-images.githubusercontent.com/54735123/107868178-05327800-6ebd-11eb-9648-34433402fe3f.png)

![image](https://user-images.githubusercontent.com/54735123/107868181-0f547680-6ebd-11eb-8a4d-97e46ab7b7f5.png)

![image](https://user-images.githubusercontent.com/54735123/107868206-3f037e80-6ebd-11eb-847c-5eb351e3b09a.png)

@damithc Hello, professor. In the situation where the binary checkbox appears as the first checkbox in a row, which design do you think is suitable?

Reply:
I'm OK with aligning right (even in cases where it appears as the last label of a row)